opm/parser/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -412,8 +414,16 @@ namespace Opm {
         }
     }
 
+    std::vector< int > EclipseState::getRegions() const {
```

Comment:
Sure. I don't know if region identifiers are pure integers or not, but Norne only uses ints so I figured it was a reasonable assumption. If it's not then I'm happy to change it.